modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java

```diff
@@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
                     }
                     outputData = process(inputData);
                     complexEventPopulater.populateComplexEvent(complexEvent, outputData);
+                    break;
```

Comment:
ok I will remove it.

On Tue, May 5, 2015 at 2:17 PM, Lasantha Fernando <notifications@github.com>
wrote:

> In
> modules/siddhi-core/src/main/java/org/wso2/siddhi/core/query/processor/stream/function/StreamFunctionProcessor.java
> <https://github.com/wso2/siddhi/pull/46#discussion_r29651644>:
>
> > @@ -42,6 +44,7 @@ protected void processEventChunk(ComplexEventChunk complexEventChunk, Processor
> >                      }
> >                      outputData = process(inputData);
> >                      complexEventPopulater.populateComplexEvent(complexEvent, outputData);
> > +                    break;
>
> Keeping it makes no harm. But technically it is not needed. +1 to remove
> if there is no other compelling reason.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wso2/siddhi/pull/46/files#r29651644>.
>



-- 
Software Engineer
WSO2 Inc.; http://wso2.com
<http://www.google.com/url?q=http%3A%2F%2Fwso2.com&sa=D&sntz=1&usg=AFQjCNEZvyc0uMD1HhBaEGCBxs6e9fBObg>
lean.enterprise.middleware

mobile: *+94728671315*